index.bs

```diff
@@ -9246,6 +9037,738 @@ The <dfn>recommended range and default for a WebAuthn ceremony timeout</dfn> is
+clientDataJSON = h'7b2274797065223a22776562617574686e2e637265617465222c226368616c6c656e6765223a223334554f436474712d392d7255576c336b5642735f446c4866783944706e776a5635543032414c49614438222c226f726967696e223a2268747470733a2f2f6578616d706c652e6f7267222c2263726f73734f726967696e223a66616c73652c22657874726144617461223a22636c69656e74446174614a534f4e206d617920626520657874656e6465642077697468206164646974696f6e616c206669656c647320696e20746865206675747572652c207375636820617320746869733a205f61366a2d747261513362436975786f7539735f62773d3d227d'
+attestationObject = h'a363666d74667061636b65646761747453746d74a263616c67266373696758473045022100a5d57ae67a212ea7e0812a37f3c692ae39062bc948132f14c433d21bfbe1bf6302203b22120bd5d958defb4ab1bd91577a1114152cbe6e31b3568881695aa6238b6a68617574684461746158a4bfabc37432958b063360d3ad6461c9c4735ae7f8edd46592a5e0f01452b2e4b54100000000b4bbfa5d68e1693b6ef5a19a0e60ef7e0020db1e8841e85c572bc6e8565f5880ae590e92f5b084c610b02900abb05974b50ca501020326200121582093f1794dea21617508ffbd343b034424782e9182774f027037bb3ff00f9ac674225820203cb763815962c1af3b82a5ef102d4e739a69d94ef639a95d486b525ab895cf'
+</xmp>
+
+[=authentication ceremony|Authentication=]:
+<xmp class="example" highlight="cddl">
+; auth_data_UV_BS sets the UV and BS bits of the authenticator data flags, but BS is set only if BE was set in the registration
+auth_data_UV_BS = h'44'   ; Derived by: HKDF-SHA-256(IKM='WebAuthn test vectors', salt=h'07', info='packed-self.ES256', L=1)
+challenge = h'1f54787ee3296594c03c12aef796c630c7510c32ca8a12927f61ac17ba82d4b5'   ; Derived by: HKDF-SHA-256(IKM='WebAuthn test vectors', salt=h'08', info='packed-self.ES256', L=32)
+client_data_gen_flags = h'81'   ; Derived by: HKDF-SHA-256(IKM='WebAuthn test vectors', salt=h'09', info='packed-self.ES256', L=1)
+; extra_client_data is included iff bit 0x01 of client_data_gen_flags is 1
+extra_client_data = h'a131beb78762e3666ee614e625001d65'   ; Derived by: HKDF-SHA-256(IKM='WebAuthn test vectors', salt=h'0a', info='packed-self.ES256', L=16)
+authenticatorData = h'bfabc37432958b063360d3ad6461c9c4735ae7f8edd46592a5e0f01452b2e4b50500000000'
+clientDataJSON = h'7b2274797065223a22776562617574686e2e676574222c226368616c6c656e6765223a224831523466754d705a5a544150424b75393562474d4d645244444c4b69684b536632477346377143314c55222c226f726967696e223a2268747470733a2f2f6578616d706c652e6f7267222c2263726f73734f726967696e223a66616c73652c22657874726144617461223a22636c69656e74446174614a534f4e206d617920626520657874656e6465642077697468206164646974696f6e616c206669656c647320696e20746865206675747572652c207375636820617320746869733a206f54472d7434646934325a753568546d4a5141645a513d3d227d'
+signature = h'3045022100a9d7fd6cddcdb9e37499bd15c9dfc27920ce6a5744c2cfdf715f13f9a275b79902201503b3c7d97b038df7a9fabd33a213318a764dc9425b93a4e9136cba863f5a09'
+</xmp>
+
+
+### ES256 Credential with "crossOrigin": true in clientDataJSON ### {#sctn-test-vectors-none-es256-crossOrigin}
```

Comment:
Indeed, see https://github.com/w3c/webauthn/pull/2209#discussion_r1860620272. I've also added a mention that no attestation is used unless otherwise noted.